zerver/lib/actions.py

```diff
@@ -2242,7 +2240,9 @@ def check_message(sender: UserProfile, client: Client, addressee: Addressee,
         elif stream_id is not None:
             stream = validate_stream_id_with_pm_notification(stream_id, realm, sender)
         else:
-            stream = addressee.stream()
+            addressee_stream = addressee.stream()
```

Comment:
I think we actually just want an `asset stream is not None` just after this else clause; I changed this to that.

Reply:
Agreed :+1: